c2corg_ui/templates/index.html

```diff
@@ -4,6 +4,7 @@
 <%inherit file="base.html"/>
 
 <div class="home-page">
+  <app-announcement></app-announcement>
```

Comment:
Just curious: wasn't it possible to add the announcement directive to the general ``base.html`` instead of doing it for every template?

Reply:
At the beginning i added the announcement directive in base.html, but the background is different in some page with the redesign, (we need to fix it in the future).

This time this way is more easy.